bin/release.sh

```diff
@@ -28,10 +28,17 @@ if [ ! "${TAG_NAME}" ] ; then
 fi
 
 # Build environment setup
-mkdir -p /tmp/gopath/src/istio.io
-ln -s /workspace /tmp/gopath/src/istio.io/pilot
-cd /tmp/gopath/src/istio.io/pilot
-touch platform/kube/config
+function prepare_gopath() {
```

Comment:
please add comment to help understand what it does.

Reply:
Done